### ISSUE TITLE
Requires -ms-transform for transition, even when it's not supported by IE

### ISSUE THREAD

**Issue by borissamardzija** [CSSLint/csslint#573]
Hello,

for example, I have following code

```css
.csstransitions .move-me {
	-webkit-transition: -webkit-transform .2s;
	-webkit-transform: translate3d(0,60px,0);
	-webkit-backface-visibility: hidden;

	-moz-transition: -moz-transform .2s;
	-moz-transform: translate3d(0,60px,0);
	-moz-backface-visibility: hidden;

	-o-transition: -o-transform .2s;
	-o-transform: translateY(60px);

	transition: transform .2s;
	transform: translateY(60px);
}
```

I am detecting browser features using Modernizr and if browser supports CSS transitions, it gets `csstransitions` class on root element, so this code applies only for browsers that Modernizr CSS transitions support.

As you can see, what it does is moving (transitioning) the element for 60px in period of 0.2s.

Vendor prefixes included (webkit, moz, o), but not for IE because it [doesn't support CSS transitions] (http://caniuse.com/#feat=css-transitions) using vendor prefix -ms (IE9).

But **CSSLint** (v0.10.0) issues a warning for this code `Warning - The property -ms-transform is compatible with -webkit-transform, -moz-transform, -o-transform and should be included as well.`

If I add `-ms-transform` property, the warning is gone, but there really is no need for it.

```css
.csstransitions .move-me {
	-webkit-transition: -webkit-transform .2s;
	-webkit-transform: translate3d(0,60px,0);
	-webkit-backface-visibility: hidden;

	-moz-transition: -moz-transform .2s;
	-moz-transform: translate3d(0,60px,0);
	-moz-backface-visibility: hidden;

	-o-transition: -o-transform .2s;
	-o-transform: translateY(60px);

	/*-ms-transition: -ms-transform .2s; not supported by IE9 */
	-ms-transform: translateY(60px); /* no need for that? */

	transition: transform .2s;
	transform: translateY(60px);
}
```

I think this warning has no sense because there is no applicable vendor prefix for `transition` in IE (-ms), so I don't need to set `transform` property in this case.

This feature [is supported without vendor prefix starting with IE10] (http://caniuse.com/#feat=css-transitions).

Is this something that can be fixed in CSSLint? I really don't like the idea turning of the `compatible-vendor-prefixes`.

Regards,
Boris.

**Comment by janaspage** [comment]
Me too.
